### ISSUE TITLE
Animation Overhaul

### ISSUE THREAD

**Issue by WestonVincze** [WestonVincze/website-overhaul#77]
## Description
Lots of feedback that the animations on the site are too long and should be skippable. Ensure that accessibility "prefers reduced motion" is respected as well.

## A/C
- Animations are shortened overall
- Animations are skippable
- prefers-reduced-motion implemented
